### PR TITLE
Minitad, Boxpol, Clowncar

### PR DESCRIPTION
**Pull request by Swagile**
## About The Pull Request

image

Also adds (1) and (2) extra sentry gun emplacements (the ones you can put on DS / tadpole) to PoS and Sulaco respectively. Fixes the issue of not being able to properly equip tadpole (old tad only had 2 sentry emplacements so this was never an issue, 2 is now mandatory and 4 is optimal).

## Why It's Good For The Game

The goal of tadpole was to be a rapid response combat bus / medevac but it was not possible to serve this purpose. After playing as PO, SL, and FC and using the current tadpole extensively, the issue is... the tadpole is too big and janky. The front of tadpole is hard to defend due to windows being wankily placed so its easy for the turrets to hit the windows themselves while trying to defend tad, its hard to manuever around front of tad to protect it due to the way the windows are positioned, and its hard to place it around cover which makes it easily crusher / acided from afar without much resistance put up.

This fixes the issue of tad being too big and bad window placement at front which now allows it to be a rapid response aircraft. Heal bus is still possible with this setup (though a bit cramped), and tadops will be a true riot to try and wrangle in this beastie.

Last but not least, even if it looks bad, I can tell you at least 3 different ways to properly compensate for the weaknesses of this new design; you just need to be a smart marine (hint: t-gas is one method).

## Changelog
:cl:
balance: Tadpole has been made into a true Clowncar, Boxpol, minitad.
balance: Extra turrets given to the TGMC for free to properly outfit either the DS on all sides or tadpole on all sides.
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
If the only thing that can fit on it is turrets, and you're giving extra turrets for it, why not just make it have only turrets by default?

**Comment by Swagile** [comment]
> 
> 
> If the only thing that can fit on it is turrets, and you're giving extra turrets for it, why not just make it have only turrets by default?

because i tried to figure it out and im either too sleep deprived, too retarded, or missing something obvious

tl;dr i dunno how to do this

**Comment by BraveMole** [comment]
Yeah but it's ugly?

**Comment by Swagile** [comment]
> 
> 
> Yeah but it's ugly?

no ones gunna care how it looks when it comes swooping in to save people's asses 😎

**Comment by Swagile** [comment]
im just gunna redo this PR cause shits fucked despite me merging and recopying maps
